views/Choropleth.svelte

```diff
@@ -74,6 +74,9 @@
           {annotations}
           {annotationRadius} />
       {/if}
+      <AnnotationsLegend
```

Comment:
@fromdusttilldawn could you add an` if(mapAnnotations && mapAnnotations.length > 0) `check here. So if no annotations are present, there is no extra margin added?

<img width="656" alt="Screenshot 2021-02-05 at 15 44 02" src="https://user-images.githubusercontent.com/1810384/107048348-21e9f400-67c9-11eb-98ea-1ead19a8d66b.png">

Reply:
Hm. If there are no annotations, then the Svelte component won't be rendered, right? And if it's not rendered, then the `margin-bottom` property won't have any effect?

Reply:
Ah wait, my bad. I didn't think it through

Reply:
Great 👍